query_reformulation.py

```diff
@@ -0,0 +1,35 @@
+from llama_index.core import Settings
+from prompt import wrap_with_entity_prompt
+import os
+import json
+import requests
+
+
+def fetch_related_taxons(query: str):
+    entity_taxon_map = {}
+    for i in range(int(os.environ["RETRY_COUNT"])):
```

Comment:
consider using `.get()` to fallback to a sensible default when fetching env variables. Also would be useful to have a `.env.dist` file with defaults for quick setup

Reply:
Makes sense, I have pushed this.